src/main/java/jmgomez/policyapi/service/PolicyService.java

```diff
@@ -23,25 +28,56 @@ public class PolicyService implements IPolicyService {
     @Autowired
     private AccidentMapper accidentMapper;
 
+    @Autowired
+    private CircuitBreakerFactory circuitBreakerFactory;
```

Comment:
Why. Why are you injecting something you are not using later. I guess you tried something that failed but you should keep your code tidy. Awful finding something like this at our current stage.

Reply:
That was a big mistake. I'm going to solve this tupe of mistakes